Kiosk/App/Views/RegisterFlowView.swift

```diff
@@ -24,10 +24,10 @@ class RegisterFlowView: ORStackView {
     }
     
     lazy var titles: Array<String> = {
-        return ["Mobile", "Email", "Credit Card"] + (self.appSetup.needsZipCode ? ["Postal/Zip"] : [])
+        return ["Mobile", "Email"] + (self.appSetup.needsZipCode ? ["Postal/Zip"] : []) + ["Credit Card"]
     }()
     lazy var keypaths: Array<Array<String>> = {
-        return [["phoneNumber"], ["email"], ["creditCardName", "creditCardType"]] + (self.appSetup.needsZipCode ? [["zipCode"]] : [])
+        return [["phoneNumber"], ["email"]] + (self.appSetup.needsZipCode ? [["zipCode"]] : []) + [["creditCardName", "creditCardType"]]
```

Comment:
Looking down below, it looks like we iterate both of these arrays simultaneously with an index `i`, which is only used to pull data out of the arrays. Could we instead make this a single array of some `struct` that encapsulates all the relevant info? Then we could just use a `forEach`. That's a pattern I've had a lot of success with in the past -- using the type system to restrict to exactly the values our logic allows and using HoF's like `map`/`forEach` to eliminate any possibility of off-by-one errors.